test/v2/unit/coins/ltc.js

```diff
@@ -154,25 +155,27 @@ describe('LTC:', function() {
     });
 
     it('should generate 3/3 custom chain p2sh-wrapped segwit address', () => {
-      const generatedAddress = coin.generateAddress({ keychains, threshold: 3, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH, chain: 20, index: 756 });
+      const addressType = Codes.UnspentTypeTcomb('p2shP2wsh');
+      const chain = Codes.forType(addressType)[Codes.PurposeTcomb('external')];
+      const generatedAddress = coin.generateAddress({ keychains, threshold: 3, addressType, chain, index: 756 });
       const generatedTestAddress = testCoin.generateAddress({
         keychains,
         threshold: 3,
-        addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH,
-        chain: 20,
+        addressType,
+        chain,
         index: 756
       });
 
       [generatedAddress, generatedTestAddress].forEach((currentAddress) => {
-        currentAddress.chain.should.equal(20);
+        currentAddress.chain.should.equal(chain);
```

Comment:
Same here, how come this needs changing?

Reply:
Same as above.